docs/resources/dms_rocketmq_instance.md

```diff
@@ -0,0 +1,178 @@
+  Changing this parameter will create a new resource.
+
+* `vpc_id` - (Required, String, ForceNew) VPC ID.
+
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Required, String, ForceNew) Subnet information.
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Required, String) ID of the security group to which the instance belongs.
+
+* `availability_zone` - (Required, List, ForceNew) Name of the AZ where instance brokers reside and which
+  has available resources.
+
+  Changing this parameter will create a new resource.
+
+* `product_id` - (Required, String, ForceNew) RocketMQ instance specifications. The options are as follows:
+  + **c6.4u8g.cluster**: maximum number of topics on each broker: 4000; maximum number of consumer groups
+    on each broker: 4000
+  + **c6.8u16g.cluster**: maximum number of topics on each broker: 8000; maximum number of consumer groups
+    on each broker: 8000
+  + **c6.12u24g.cluster**: maximum number of topics on each broker: 12,000; maximum number of consumer groups
+    on each broker: 12,000
+  + **c6.16u32g.cluster**: maximum number of topics on each broker: 16,000; maximum number of consumer groups
+    on each broker: 16,000
+  Changing this parameter will create a new resource.
+
+* `storage_spec_code` - (Required, String, ForceNew) Storage I/O specification. The options are as follows:
+  + **dms.physical.storage.high.v2**: high I/O disk
+  + **dms.physical.storage.ultra.v2**: ultra-high I/O disk
+  Changing this parameter will create a new resource.
+
+* `description` - (Optional, String) Instance description.
+  The description can contain a maximum of 1024 characters.
+
+* `ssl_enable` - (Optional, Bool, ForceNew) Whether to enable SSL encryption for access.
+  Changing this parameter will create a new resource.
+
+* `ipv6_enable` - (Optional, Bool, ForceNew) Whether to support IPv6.
+  Changing this parameter will create a new resource.
+
+* `enable_publicip` - (Optional, Bool, ForceNew) Whether to enable public access.
+  By default, public access is disabled.
+  Changing this parameter will create a new resource.
+
+* `publicip_id` - (Optional, String, ForceNew) ID of the EIP bound to the instance.
+  Use commas (,) to separate multiple EIP IDs.
+  This parameter is mandatory if public access is enabled (that is, enable_publicip is set to true).
+  Changing this parameter will create a new resource.
+
+* `broker_num` - (Optional, Int, ForceNew) Number of brokers.
+  Changing this parameter will create a new resource.
+
+* `retention_policy` - (Optional, Bool) ACL.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `status` - MStatus.
+
+* `type` - Instance type. Value: cluster.
+
+* `specification` - Instance specification.
+
+* `instance_id` - Instance ID.
+
+* `charging_mode` - Billing mode. 1 indicates pay-per-use.
```

Comment:
useless attribute.
And this value is not `1`, it should be `postPaid`.

Reply:
done